app/models/testtrack_cli.rb

```diff
@@ -0,0 +1,35 @@
+require 'rbconfig'
+
+class TesttrackCli
+  include Singleton
+
+  def project_initialized?
+    File.exist?(File.join('testtrack', 'schema.yml'))
```

Comment:
Should this look at `Rails.root`?

Reply:
I was gonna have it do that but then I’d need to pushd into it to run tt cli, I futzed with that for a while and it was getting annoying (ruby stdlib Shell was logging nastiness out to the screen), so I figured this is symmetrical, and requires none of that.

Reply:
I don't follow. Why not do Rails.root.join here?

Reply:
Because ultimately if the user isn’t in the app root, testtrack cli is going to fail, so unless I wire it up to work when calling rake from a random non-app-root directory, I might as well not app-root-bind the schema lookup. It’s gonna have a bad time if it thinks it’s setup and then can’t migrate or schema load